internal/proto/local/controller/api/resources/accounts/v1/account.proto

```diff
@@ -14,83 +14,83 @@ import "controller/custom_options/v1/options.proto";
-		];
+		];  // @gotags: `class:"public"`
 	}
 
 	// Output only. managed_group_ids indicates IDs of the managed groups that currently contain this account
-	repeated string managed_group_ids = 110 [json_name="managed_group_ids"];
+	repeated string managed_group_ids = 110 [json_name="managed_group_ids"];  // @gotags: `class:"public"`
 
 	// Output only. The available actions on this resource for this user.
-	repeated string authorized_actions = 300 [json_name="authorized_actions"];
+	repeated string authorized_actions = 300 [json_name="authorized_actions"];  // @gotags: `class:"public"`
 }
 
 // Attributes associated only with Accounts with type "password".
 message PasswordAccountAttributes {
 	// The login name of this Account. This is unique per Auth Method.
-	string login_name = 10 [json_name="login_name", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.login_name" that: "LoginName"}];
+	string login_name = 10 [json_name="login_name", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.login_name" that: "LoginName"}];  // @gotags: `class:"sensitive"`
 
 	// The password for this Account.
-	google.protobuf.StringValue password = 20 [(custom_options.v1.generate_sdk_option) = true];
+	google.protobuf.StringValue password = 20 [(custom_options.v1.generate_sdk_option) = true];  // @gotags: `class:"secret"`
 }
 
 // Attributes associated only with Accounts with type "oidc".
 message OidcAccountAttributes {
 	// issuer is a case sensitive URL that maps to the OIDC iss claim.
 	// This value is immutable after creation time.
-	string issuer = 80 [json_name="issuer", (custom_options.v1.generate_sdk_option) = true];
+	string issuer = 80 [json_name="issuer", (custom_options.v1.generate_sdk_option) = true];  // @gotags: `class:"sensitive"`
 
 	// subject is a case sensitive string that maps to the OIDC sub claim.
 	// This value is immutable after creation time.
-	string subject = 90 [json_name="subject", (custom_options.v1.generate_sdk_option) = true];
+	string subject = 90 [json_name="subject", (custom_options.v1.generate_sdk_option) = true];  // @gotags: `class:"sensitive"`
 
 	// Output only. full_name is a string that maps to the OIDC name claim.
-	string full_name = 100 [json_name="full_name"];
+	string full_name = 100 [json_name="full_name"];  // @gotags: `class:"sensitive"`
 
 	// Output only. email is a string that maps to the OIDC email claim.
-	string email = 110;
+	string email = 110;  // @gotags: `class:"sensitive"`
 
 	// Output only. token_claims are the marshaled claims from the token.
-	google.protobuf.Struct token_claims = 120;
+	google.protobuf.Struct token_claims = 120;  // @gotags: `class:"sensitive"`
 
 	// Output only. userinfo_claims are the marshaled claims from userinfo.
-	google.protobuf.Struct userinfo_claims = 130;
+	google.protobuf.Struct userinfo_claims = 130;  // @gotags: `class:"sensitive"`
```

Comment:
I marked all of these as sensitive because they could contain PII. Do we consider `subject` a secret?